modules/swagger-codegen/src/main/resources/Javascript/es6/model_test.mustache

```diff
@@ -1,19 +1,12 @@
 {{>licenseInfo}}
+import expect, { createSpy, spyOn, isSpy } from 'expect';
+
 (function(root, factory) {
-  if (typeof define === 'function' && define.amd) {
-    // AMD.
-    define(['expect.js', '../../src/index'], factory);
-  } else if (typeof module === 'object' && module.exports) {
-    // CommonJS-like environments that support module.exports, like Node.
-    factory(require('expect.js'), require('../../src/index'));
-  } else {
-    // Browser globals (root is window)
-    factory(root.expect, root.{{moduleName}});
-  }
+
```

Comment:
How is this going to work? The module will be empty since the `factory` isn't called.